File: Sveikieji_ir_realieji_skaiciai/Praktika/Uzduotis_namuose_sprendimas.cpp
#include <iostream>
using namespace std;

// Funkcija, kuri konvertuoja sveikąjį skaičių į realųjį
float int_to_float(int n) {
    return static_cast<float>(n);
}

// Funkcija, kuri konvertuoja realųjį skaičių į sveikąjį
int float_to_int(float n) {
    return static_cast<int>(n);
}

int main() {
    int sveikas_skaicius;
    float realus_skaicius;

    // Vartotojo įvedimas
    cout << "Įveskite sveikąjį skaičių: ";
    cin >> sveikas_skaicius;

    cout << "Įveskite realųjį skaičių: ";
    cin >> realus_skaicius;

    // Konvertavimas ir rezultatų spausdinimas
    cout << "Sveikasis skaičius konvertuotas į realųjį: " << int_to_float(sveikas_skaicius) << endl;
    cout << "Realusis skaičius konvertuotas į sveikąjį: " << float_to_int(realus_skaicius) << endl;

    return 0;
}

File: Sveikieji_ir_realieji_skaiciai/Praktika/Uzduotis_namuose_sprendimas.py
# Funkcija, kuri konvertuoja sveikąjį skaičių į realųjį
def int_to_float(n):
    return float(n)

# Funkcija, kuri konvertuoja realųjį skaičių į sveikąjį
def float_to_int(n):
    return int(n)

# Vartotojo įvedimas
sveikas_skaicius = int(input("Įveskite sveikąjį skaičių: "))
realus_skaicius = float(input("Įveskite realųjį skaičių: "))

# Konvertavimas ir rezultatų spausdinimas
print(f"Sveikasis skaičius konvertuotas į realųjį: {int_to_float(sveikas_skaicius)}")
print(f"Realusis skaičius konvertuotas į sveikąjį: {float_to_int(realus_skaicius)}")
